src/pages/ItemsPage.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import { getItems } from "../api/ItemApi";
+import PandaBestList from "../components/PandaBestList.jsx";
+import PandaAllList from "../components/PandaAllList.jsx";
+
+function ItemsPage() {
+  const [items, setItems] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const data = await getItems();
+        setItems(data);
+        setLoading(false);
+      } catch (err) {
+        setError(err.message);
+        setLoading(false);
+      }
```

Comment:
setLoading 은 api 의 에러 여부와 관계없이 처리되는 값이라서 finally 로 받아주는 게 자연스러울 거 같아요!
```suggestion
      try {
        const data = await getItems();
        setItems(data);
      } catch (err) {
        setError(err.message);
      } finally {
        setLoading(false);
      }
```